src/ingest_validation_tools/table-schemas/level-2/scrnaseq.yaml

```
@@ -103,6 +103,8 @@ fields:
   -
     name: sequencing_read_format
     description: Number of sequencing cycles in Read1, i7 index, i5 index, and Read2
+    constraints:
+     pattern: '[Ri]?\d+(/[Ri]?\d+)+'
```

```suggestion
    description: 'Number of sequencing cycles in Read1, i7 index, i5 index, and Read2 - TODO https://github.com/hubmapconsortium/ingest-validation-tools/issues/303'
```
I don't think that regex does what we want.